flycheck-ledger.el

```diff
@@ -46,7 +46,8 @@
           (zero-or-more line-start (or "While " "> ") (one-or-more not-newline) "\n" )
           (message (minimal-match (zero-or-more line-start (zero-or-more not-newline) "\n"))
                    "Error: " (one-or-more not-newline) "\n")))
-  :modes ledger-mode)
+  :modes ledger-mode
+  :next-checkers (ledger-zero))
```

Comment:
You could prevent `ledger-zero` from running if there were any errors reported by `ledger` with `:next-checkers (warning . ledger-zero)`.  You might be aware of this; I don't know whether you intended to run it unconditionally.